ocs_ci/helpers/helpers.py

```
@@ -7669,3 +7671,220 @@ def delete_cephfs_ec_pool(pool_name):
+
+def setup_scale_cluster_infrastructure_for_cnsa_rm(
+    request, namespace=constants.IBM_STORAGE_SCALE_NAMESPACE
+):
+    """
+    Deploys core Scale operator (MCO), Entitlement key, and local IBM Scale Cluster CR.
+    Registers a finalizer on the request fixture to guarantee proper resource scrubbing on teardown.
+    """
+    logger.info("--- Setup: IBM Storage Scale Infrastructure ---")
+    cluster_name = "ibm-spectrum-scale"
+
+    def finalizer_cleanup():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `finalizer_cleanup`

Add return type annotation: `None`

(ANN202)